src/graphql/mutations/hooks/patients.tsx

```diff
@@ -0,0 +1,95 @@
+import { useMutation } from "@apollo/react-hooks";
+import { ADD_PATIENT, EDIT_PATIENT } from "../templates/patients";
+import { CCPType } from "../../queries/templates/collectionPoints";
+import { triageLevel, status } from "../../queries/templates/patients";
+
+export function addPatientMutation(
```

Comment:
Correct me if i'm wrong, but I think this `addPatientMutation` function is unused? Please remove it

Reply:
I believe hooks for events are also unused, we should probably remove hooks for mutations and just use the useMutation hook directly for all mutations